pkgs/applications/misc/spacenav-cube-example/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, libspnav, libX11, mesa_glu}:
+
+stdenv.mkDerivation {
+  pname = "spacenav-cube-example";
```

Comment:
I think it would make a lot of sense to put the binary from this example package into the library. It's not really a package anybody would want to depend on, it's just like an add-on.

Reply:
I've extracted this into its own derivation after seeing the discussion in https://github.com/NixOS/nixpkgs/pull/12402
I'll be happy to change this, if there is consensus that that will be merged.

Reply:
Ah I see, thanks for linking that, sounds reasonable.